ddlambda.go

```diff
@@ -92,6 +93,21 @@ const (
 	DefaultEnhancedMetrics = true
 )
 
+// WrapHandlerInterface is used to instrument your lambda functions.
+// It returns a modified handler that can be passed directly to the lambda.StartHandler function.
+func WrapHandlerInterface(handler lambda.Handler, cfg *Config) lambda.Handler {
```

Comment:
What was your thought process in terms of naming this `WrapHandlerInterface`?

Also, it looks like this function is 99% the same as `WrapHandler`? Is there an opportunity to re-use some code?

Reply:
The name is tricky. The other method takes an interface, but is named WrapHandler already. Whereas this one takes a Handler interface. I couldn't think of a good name, but I'm up for any suggestions

Reply:
Whatever we land on, I think in that case `WrapHandlerInterface` could be especially confusing because like you said, it's the other method that takes an interface.

Reply:
Maybe `WrapLambdaHandler`? Since the type it takes is `lambda.Handler`?

I would also consider expanding the doc comments for these two methods to make sure there is no confusion between them.